exporter/prometheusexporter/collector.go

```diff
@@ -20,7 +20,7 @@ import (
 	"go.uber.org/zap"
 	"google.golang.org/protobuf/proto"
 
-	prometheustranslator "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus"
+	prometheustranslator "github.com/prometheus/otlptranslator"
```

Comment:
I don't see the need for renaming the import? With the old package, it made sense as it's just named "prometheus".
```suggestion
	"github.com/prometheus/otlptranslator"
```